spec/karma.conf.js

```diff
@@ -22,6 +22,7 @@ module.exports = (config) => {
     // list of files / patterns to load in the browser
     files: [
       'javascripts/angular_cache.js',
+      'javascripts/vendor.js',
```

Comment:
it contains underscore dependency, so had to list it here explicitly to load before the main app code.